src/css/Properties.js

```diff
@@ -460,7 +460,7 @@ var Properties = module.exports = {
     // U
     "unicode-bidi"                      : "normal | embed | isolate | bidi-override | isolate-override | plaintext",
     "user-modify"                       : "read-only | read-write | write-only",
-    "user-select"                       : "none | text | toggle | element | elements | all",
+    "user-select"                       : "none | auto | text | all | contain | element | elements | toggle",
```

Comment:
That draft has been superseded. The current draft is https://drafts.csswg.org/css-ui-4/#propdef-user-select .

I can't find `toggle` in any of the more current drafts.